### DISCUSSION TITLE
FastApi docs. Terminal output integration

### DISCUSSION THREAD

**Discussion by ElAouaneHamza** [fastapi/fastapi#8649] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.get("/azure")
def az_cli (args_str):
    temp = tempfile.TemporaryFile()
    args = args_str.split()
    code = get_default_cli().invoke(['login', '--service-principal', '-u', '', '-p', '','--tenant',''])
    resource = get_default_cli().invoke(args)
    data = temp.read().strip()
    temp.close()
    return [args, resource]
```

### Description

sorry for this basic question but I would like some help from you expert as I am still learning fastaapi.

I have a simple testing application running python FastApi and trying to use it with azure cli.

what I am trying to do, is to have a get request using fastapi to list all the resource groups I have in my subscriptions.

The Above code works just fine, I am able to run it as expected, but the output is visible in my terminal only, in `/docs` all what I receive back is the code 200 and that's it. 

I know this question might sound very basic but I really don't know exactly where to look and how to add the terminal output to my response body.

Thank you very much for any help you can provide me with and I am sorry for the basic question

### Operating System

macOS

### FastAPI Version

0.68.1

### Python Version

3.9.6

**Comment by masoodmomin** [comment, marked as answer]
Its not related to FastAPI. But I found a solution for you [here](https://stackoverflow.com/questions/52340497/how-to-save-the-output-of-azure-cli-commands-in-a-variable).

**Comment by STeveShary** [comment]
@ElAouaneHamza With you example, it's a bit difficult to understand what the data is that you want to return from the function.  I would recommend that you look at building a JSONResponse where you can craft the exact data that you want to return: https://fastapi.tiangolo.com/advanced/response-directly/#using-the-jsonable_encoder-in-a-response

The alternative is to create a pydantic model object and rather than return a list, you should return a pydantic object.  A good example on returning a pydantic object can be found here: https://fastapi.tiangolo.com/tutorial/response-model/
